### PR TITLE
IEEE paper edits 

### PR DESCRIPTION
**Pull request by Prashanthsrn**
### 1. Added Model Evaluation Metrics
  - Binary cross entropy loss
  - MAE
  - Time taken to train model
 

### 2. Standardized Hyperparameters
  - Epochs: 100
  - Batch size: 16

### 3. Documentation Updates
- Added recently submitted papers to `MidcurveNN/publications/MidcurveNN_ICCCIT2025/` directory

### CODE REVIEW

**Review by yogeshhk — APPROVED** [review]
Good work
